### PR TITLE
chore: store wallet stuff inside a sub-dir

### PR DESCRIPTION
**Pull request by bonomat**
That's a bit cleaner in the directory

### CODE REVIEW

File: rust/src/wallet.rs [hunk 1]
```diff
@@ -100,7 +100,7 @@ impl Wallet {
         )?;
 
         // Create a database (using default sled type) to store wallet data
-        let db = bdk::sled::open(data_dir.clone())?;
+        let db = bdk::sled::open(data_dir.join("wallet").clone())?;
```
Comment (luckysori) [inline rust/src/wallet.rs:103]: Is this a breaking change for the maker?

Comment (bonomat) [reply rust/src/wallet.rs:103]: yes, we'd need to move the seed files. Let's not do it then for now :)

**Comment by bonomat** [comment]
Not worth the hassle

File: rust/src/wallet.rs (hunk 1 above)
Comment (bonomat) [reply rust/src/wallet.rs:103]: actually no :D

**Comment by bonomat** [comment]
> Not worth the hassle

Actually no hassle at all, there is nothing to do. Worst case we have the wallet db twice if we don't clean up.

**Review by luckysori — APPROVED** [review]

**Pull request merged**